### ISSUE TITLE
Twitter error bug

### ISSUE THREAD

**Issue by JacobT1997** [fkk-cz/noire-public#793]
This happens after I open the twitter app through the phone in-game. I would like a fix for this error because I would like to use the app when I play. If it doesn't get fixed I will not play on the server until further notice.
It can also show cannot read property 'twitter' of null as well.

![Screenshot (64)](https://user-images.githubusercontent.com/104861548/166944272-8ff8ec30-e3fd-4271-85bf-9873c122e512.png)

**Comment by PreacherDeimos** [comment]
I've heard other people having this problem, good luck.

**Comment by No-Yes1** [comment]
I believe this is caused from having the same in city name as someone, and twitter names are based from character names. @JacobT1997, What is your Characters first and last name?

**Comment by JacobT1997** [comment]
I don’t remember I’ll have to check what it is in a couple days

**Comment by JacobT1997** [comment]
My characters named Jacob Johnson.

**Comment by JacobT1997** [comment]
So is my in-game name (Jacob Johnson) the reason it’s giving me the error from the picture above?

**Comment by fkk-cz** [comment]
Yes, the issue is caused by not allowing duplicate Twitter usernames. This will be fixed once the phone is updated to the latest build.

**Comment by JacobT1997** [comment]
Can I ask if you know when the latest build will be out?

**Comment by JacobT1997** [comment]
sorry that I ask a lot of questions but can I ask when the phone will be updated to the latest build?

**Comment by JacobT1997** [comment]
If I can would changing my characters name fix the error or do I have to wait for the phones to be updated

**Comment by fkk-cz** [comment]
> sorry that I ask a lot of questions but can I ask when the phone will be updated to the latest build?

Probably when I conduct the next maintenance (in 1-2 weeks?) along with the server upgrade.

**Comment by fkk-cz** [comment]
@JacobT1997 can you confirm this is fixed on your end?

**Comment by JacobT1997** [comment]
I can use twitter
